model/src/cfd.rs

```diff
@@ -16,6 +17,7 @@ use crate::OpeningFee;
 use crate::Percent;
 use crate::Position;
 use crate::Price;
+use crate::RolloverVersion;
```

Comment:
🔧 I don't think we need this type. It should be enough to use `rollover::Version` in this file.